default/tests/shared/query_engine/expressions/test_dataset.py

```diff
@@ -35,15 +35,46 @@ def setUp(self):
         self.auth_api = common_actions.create_project_auth(project = self.project, session = self.session)
         self.member = self.auth_api.member
 
-    def test_build_expression_subquery(self):
+    def initialize_compare_expression(self):
```

Comment:
Not sure if its common convention to define helper methods as part of the test class?